trade-finance-manager-ui/server/helpers/getFlashSuccessMessage.js

```diff
@@ -0,0 +1,3 @@
+const getFlashSuccessMessage = (req) => req.flash('successMessage')[0];
```

Comment:
File copied exactly over from `portal-ui`

Reply:
Would be awesome to add some documentation to both functions whilst we're here 🙏

Reply:
Alternatively what do we think about putting it in libs/common? its very front-end oriented but I think thats ok?

Reply:
Documentation added! I'm going to leave it in their respective directories if that's ok @MarRobSoftwire - if I move it to libs/common we'd want the helper in TS which would mean installing extra dev dependencies with the `flash` types in libs/common

Reply:
Resolving as docs added👍